modules/declarative-configuration-examples.adoc

```diff
@@ -103,7 +103,7 @@ rules:
     clusterLabelSelectors:
         - requirements:
             key: kubernetes.io/metadata.name
-            operator: IN
+            operator: IN <3>
```

Comment:
I am not seeing this call out in the preview. Am I looking in the wrong place? https://docs.openshift.com/acs/4.1/configuration/declarative-configuration-using.html#declarative-config-example-access-scope